mail_tracking/models/mail_thread.py

```diff
@@ -30,43 +30,58 @@ def _get_failed_message_domain(self):
     def message_post(self, *args, **kwargs):
         """Adds CC recipient to the message.
 
-        Because Odoo implementation avoid store cc recipients we ensure that
-        this information its written into the mail.message record.
+        Because Odoo implementation avoid store 'cc/to' recipients we ensure
+        that this information its written into the mail.message record.
         """
         new_message = super().message_post(*args, **kwargs)
         email_cc = kwargs.get('cc')
+        to_write = {}
         if email_cc:
-            new_message.sudo().write({
-                'email_cc': email_cc,
-            })
+            to_write['email_cc'] = email_cc
+        # This is here because Odoo only store recipients that have a partner.
+        to_write['email_to'] = kwargs.get('to')
+        new_message.sudo().write(to_write)
         return new_message
 
     @api.multi
     def message_get_suggested_recipients(self):
-        """Adds email Cc recipients as suggested recipients.
+        """Adds email 'extra' recipients as suggested recipients.
 
         If the recipient has a res.partner, use it.
         """
         res = super().message_get_suggested_recipients()
+        db_aliases = set(map(
+            lambda x: x['alias_name'],
+            self.env['mail.alias'].search_read([
+                ('alias_name', '!=', False),
+            ], ['alias_name'])))
```

Comment:
This is the same but more readable:

```suggestion
        db_aliases = set(
            x['alias_name'] for x in
            self.env['mail.alias'].search_read([
                ('alias_name', '!=', False),
            ], ['alias_name']))
        )
```

Reply:
Not only more readable, this increase the performance! thx!